### PR TITLE
[jebbyk] Create a simple console rasteriser

### PR DESCRIPTION
**Pull request by jebbyk**
### Description
- Simple geometry rasteriser with ability to draw points, lines, triangles and filled triangles

Screenshot from 2019-07-08 17-23-37

### Context
Link to the issue: jebbyk's homework

### What has been done
- Added rasteriser to modules
- Integrated module logic to `main.py`

### Test plan
- try to draw points, lines and triangles _in positions out of the buffer_
- try to create buffer with `0` size
- try to create buffer with different `m` and `n`
- try to draw `0` length lines and `0` square figures

### Todo/notes
- testing
- convert 3d triangles to 2d space
- printing arrays of triangles as 3d models

### CODE REVIEW

File: modules/baseRasteriser.py [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+    #start = Point(10,10)
+    #end = Point(45, 56)
+
+    buffer = createBuffer(48, 48)
+
+    putPoint(buffer, 6, 16)
+
+    putLine(buffer, 4, 56, 48, 16)
+
+    # putLine(buffer, 2,2, 30,2)
```
Comment (Sinu5oid) [inline modules/baseRasteriser.py]: Remove commented lines and refactor your code

File: main.py [hunk 2]
```diff
@@ -8,4 +9,20 @@
     print(list)
     print(sinu5oid.bubbleSort(list))
     # and above here
+   
+    print('\n')
```
Comment (Sinu5oid) [inline main.py]: Move these header prints to module function just to not bloat prints here

File: modules/baseRasteriser.py (hunk 1 above)
Comment (jebbyk) [reply modules/baseRasteriser.py]: fixed

File: main.py (hunk 2 above)
Comment (jebbyk) [reply main.py]: fixed

**Pull request merged**